testsuite/MDAnalysisTests/analysis/test_hole2.py

```diff
@@ -403,7 +404,10 @@ def test_plot_mean_profile(self, hole, frames, profiles):
         binned, bins = hole.bin_radii(bins=100)
         mean = np.array(list(map(np.mean, binned)))
         stds = np.array(list(map(np.std, binned)))
-        midpoints = 0.5 * bins[1:] + bins[:-1]
+        midpoints = 0.5 * bins[1:] + 0.5 * bins[:-1]
+        # assume a bin of length 1.5
+        assert_allclose(midpoints, bins[1:], atol=0.75)
```

Comment:
This is a really creative use of `assert_allclose`! However, it is usually to better to be much more direct with tests. For example, instead of allowing any value within 0.75 of a midpoint in either direction, you could directly calculate the difference between `midpoint` and `bins` and check that it is 0.

```suggestion
        difference_right = midpoints - bins[1:]
        assert_allclose(difference_right, 0)
```

A similar test could be done with the left-hand-side (`bins[:-1]`) if you wanted to check that the midpoint was actually midway through.